### DISCUSSION TITLE
Unable to write द्वितीय in PDF

### DISCUSSION THREAD

**Discussion by ccpplinux** [mpdf/mpdf#1881] [General]
When I am trying to write द्वितीय in PDF using mpdf library then it is being displayed as दिवतीय. How to fix this issue? Please help me in this regard.

**Comment by ssieb** [comment]
The example you provided in the issue you filed is not reproducible by anyone else.  You have includes and database queries.  You need to provide a **minimal** example with just enough content to reproduce the issue.

**Comment by ccpplinux** [comment]
`<?php
require_once __DIR__ . '/mpdf/vendor/autoload.php';

$mpdf = new \Mpdf\Mpdf();
$mpdf->autoScriptToLang = true;
$mpdf->baseScript = 1;
$mpdf->autoVietnamese = true;
$mpdf->autoArabic = true;
$mpdf->autoLangToFont = true;

$html=
"
<html>
    <head>
        <style>
            body { font-family: sans; text-align: justify; }
            p { font-family: sans; }
            div { font-family: sans; }
            h1 { font-family: sans; }
            td { font-family: sans; }
        </style>
    </head>
    <body>
        Result is द्वितीय
    </body>
</html>
";
$mpdf->WriteHTML($html);
$mpdf->Output();
?>
`

**Reply by ssieb** [reply]
I have no idea what the difference is between those two texts, but maybe you have to provide the correct font.

**Comment by ccpplinux** [comment]
This is the minimal code.

**Comment by ccpplinux** [comment]
You can view the output of the script at https://www.yogile.com/q8b3y9j4260/21t/share/?vsc=9985f0b97

**Comment by finwe** [comment]
there are other bug reports concerning hindi (?) output with unnecessary characters and/or placeholders. This is most likely the bug of the library, see #1882
